### PR TITLE
[zh] sync/pods/pod-lifecycle.md

### PR DESCRIPTION
**Pull request by 0xff-dev**
[zh] sync/pods/pod-lifecycle.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
 如果容器配置了 `preStop` 回调，则该回调会在容器进入 `Terminated`
 状态之前执行。
 
+<!--## How Pods handle problems with containers {#container-restarts}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:291]:
```suggestion
<!--
## How Pods handle problems with containers {#container-restarts}
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 2]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+
+Kubernetes manages container failures within Pods using a [`restartPolicy`](#restart-policy) defined in the Pod `spec`. This policy determines how Kubernetes reacts to containers exiting due to errors or other reasons, which falls in the following sequence:
+-->
+
+## Pod 如何处理容器问题 {#container-restarts}
+
+Kubernetes 在 Pod `spec` 中定义的 [`restartPolicy`](#restart-policy) 管理 Pod 内的容器故障。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:298]:
```suggestion
Kubernetes 通过在 Pod `spec` 中定义的 [`restartPolicy`](#restart-policy) 管理 Pod 内容器出现的失效。 
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 3]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+   backoff delay for subsequent restarts, described in [`restartPolicy`](#restart-policy).
+   This prevents rapid, repeated restart attempts from overloading the system.
+1. **CrashLoopBackOff state**: This indicates that the backoff delay mechanism is currently
+   in effect for a given container that is in a crash loop, failing and restarting repeatedly.
+1. **Backoff reset**: If a container runs successfully for a certain duration
+   (e.g., 10 minutes), Kubernetes resets the backoff delay, treating any new crash
+   as the first one.
+-->
+
+1. **Initial crash**：Kubernetes 尝试根据 Pod `restartPolicy` 立即重新启动。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:313]:
```suggestion
1. **最初的崩溃**：Kubernetes 尝试根据 Pod 的 `restartPolicy` 立即重新启动。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 4]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+1. **Repeated crashes**：在 initial crash 后，对于后续的重新启动 Kubernetes 采用指数级回退延迟机制
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:314]:
```suggestion
1. **反复的崩溃**：在最初的崩溃之后，Kubernetes 对于后续重新启动的容器采用指数级回退延迟机制，
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 5]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+    如 [`restartPolicy`](#restart-policy) 中所述。
+    这可以防止快速、重复的重新启动尝试导致系统过载。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:316]:
```suggestion
    这一机制可以防止快速、重复的重新启动尝试导致系统过载。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 6]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+1. **CrashLoopBackOff state**：这表明，对于一个处于崩溃循环、反复失败和重启的给定容器，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:317]:
```suggestion
1. **CrashLoopBackOff 状态**：这一状态表明，对于一个给定的、处于崩溃循环、反复失效并重启的容器，
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 7]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+    回退延迟机制目前正在生效。
+1. **Backoff reset**：如果容器成功运行了一定时间（如 10 分钟），
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:319]:
```suggestion
1. **回退重置**：如果容器成功运行了一定时间（如 10 分钟），
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 8]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+    Kubernetes 会重置回退延迟机制，将任何新的崩溃视为第一次崩溃。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:320]:
```suggestion
    Kubernetes 会重置回退延迟机制，将新的崩溃视为第一次崩溃。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 9]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+
+<!--
+In practice, a `CrashLoopBackOff` is a condition or event that might be seen as output
+from the `kubectl` command, while describing or listing Pods, when a container in the Pod
+fails to start properly and then continually tries and fails in a loop.
+-->
+实际上，`CrashLoopBackOff` 是在描述或列出 Pod 时从 `kubectl` 命令输出的一种条件或事件，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:327]:
```suggestion
在实际部署中，`CrashLoopBackOff` 是在描述或列出 Pod 时从 `kubectl` 命令输出的一种状况或事件。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 10]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+当 Pod 中的容器无法正常启动，并在循环中不断尝试和失败时就会出现。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:328]:
```suggestion
当 Pod 中的容器无法正常启动，并反复进入尝试与失败的循环时就会出现。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 11]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+
+<!--
+In other words, when a container enters the crash loop, Kubernetes applies the
+exponential backoff delay mentioned in the [Container restart policy](#restart-policy).
+This mechanism prevents a faulty container from overwhelming the system with continuous
+failed start attempts.
+-->
+换句话说，当容器进入崩溃循环时，Kubernetes 会应用 [Container restart policy](#restart-policy)
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:336]:
```suggestion
换句话说，当容器进入崩溃循环时，Kubernetes 会应用[容器重启策略](#restart-policy) 
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 12]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+当 Pod 中的容器无法正常启动，并在循环中不断尝试和失败时就会出现。
+
+<!--
+In other words, when a container enters the crash loop, Kubernetes applies the
+exponential backoff delay mentioned in the [Container restart policy](#restart-policy).
+This mechanism prevents a faulty container from overwhelming the system with continuous
+failed start attempts.
+-->
+换句话说，当容器进入崩溃循环时，Kubernetes 会应用 [Container restart policy](#restart-policy) 
+中提到的指数级回退延迟机制。这种机制可以防止有问题的容器因不断尝试启动失败而导致系统不堪重负。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:337]:
```suggestion
中提到的指数级回退延迟机制。这种机制可以防止有问题的容器因不断进行启动失败尝试而导致系统不堪重负。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 13]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+* Configuration errors, such as incorrect environment variables or missing
+  configuration files.
+* Resource constraints, where the container might not have enough memory or CPU
+  to start properly.
+* Health checks failing if the application doesn't start serving within the
+  expected time.
+* Container liveness probes or startup probes returning a `Failure` result
+  as mentioned in the [probes section](#container-probes).
+-->
+下列问题可能会导致 `CrashLoopBackOff`：
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:352]:
```suggestion
下列问题可以导致 `CrashLoopBackOff`：
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 14]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+1. **Check resource limits**: Make sure that the container has enough CPU
+   and memory allocated. Sometimes, increasing the resources in the Pod definition
+   can resolve the issue.
+1. **Debug application**: There might exist bugs or misconfigurations in the
+   application code. Running this container image locally or in a development
+   environment can help diagnose application specific issues.
+-->
+要调查 `CrashLoopBackOff` 问题的根本原因，用户可以：
+
+1. **检查日志**： 使用 `kubectl logs <pod名称>`检查容器的日志。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:379]:
```suggestion
1. **检查日志**：使用 `kubectl logs <pod名称>` 检查容器的日志。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 15]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+    这通常是诊断导致崩溃的问题的最直接方法。
+1. **检查事件**： 使用 `kubectl describe pod <pod名称>`查看 Pod 的事件，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:381]:
```suggestion
1. **检查事件**：使用 `kubectl describe pod <pod名称>` 查看 Pod 的事件，
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 16]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+    这可以提供有关配置或资源问题的提示。
+1. **审查配置**： 确保 Pod 配置正确无误，包括环境变量和挂载卷，并且所有必需的外部资源都可用。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:383]:
```suggestion
1. **审查配置**：确保 Pod 配置正确无误，包括环境变量和挂载卷，并且所有必需的外部资源都可用。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 17]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+1. **检查资源限制**： 确保容器分配了足够的 CPU 和内存。有时，增加 Pod 定义中的资源可以解决问题。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:384]:
```suggestion
1. **检查资源限制**： 确保容器被分配了足够的 CPU 和内存。有时，增加 Pod 定义中的资源可以解决问题。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 18]
```diff
@@ -288,6 +288,103 @@ the `Terminated` state.
...
+1. **调试应用程序**： 应用程序代码中可能存在错误或配置不当。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:385]:
```suggestion
1. **调试应用程序**：应用程序代码中可能存在错误或配置不当。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 19]
```diff
@@ -316,20 +417,25 @@ Pod 级别的 `restartPolicy` 字段：在 Kubernetes 中，Sidecar 被定义为
 对于因错误而退出的 Init 容器，如果 Pod 级别 `restartPolicy` 为 `OnFailure` 或 `Always`，
 则 kubelet 会重新启动 Init 容器。
 
+* `Always`： 在任何终止后自动重启容器。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:420]:
```suggestion
* `Always`：只要容器终止就自动重启容器。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 20]
```diff
@@ -316,20 +417,25 @@ Pod 级别的 `restartPolicy` 字段：在 Kubernetes 中，Sidecar 被定义为
...
+* `OnFailure`： 只有在容器错误退出（退出状态非零）时才重新启动容器。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:421]:
```suggestion
* `OnFailure`：只有在容器错误退出（退出状态非零）时才重新启动容器。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 21]
```diff
@@ -316,20 +417,25 @@ Pod 级别的 `restartPolicy` 字段：在 Kubernetes 中，Sidecar 被定义为
...
+* `Never`： 不会自动重启已终止的容器。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:422]:
```suggestion
* `Never`：不自动重启已终止的容器。
```

**Review by windsonsea — COMMENTED** [review]
See preview: https://deploy-preview-46288--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/

**Pull request merged**